requirements.txt

```diff
@@ -24,3 +24,4 @@ mecab-python3==1.0.3
 unidic-lite==1.0.8
 # gruut+supported langs
 gruut[cs,de,es,fr,it,nl,pt,ru,sv]~=1.2.0
+fsspec
```

Comment:
can you tag the working version too ?

Reply:
definitely, but since these requirements are specified when installing as a dependency how about setting a lower bound?